jpasskit/src/main/java/de/brendamour/jpasskit/signing/PKPassTemplateInMemory.java

```diff
@@ -136,15 +136,14 @@ public void addAllFiles(String directoryWithFilesToAdd) throws IOException {
         }
     }
 
-    public Map<String, InputStream> getFiles() {
+    public Map<String, byte[]> getFiles() {
```

Comment:
I am not sure if this is considered as api change.
I guess the reason behind this method is just testing, but since it is public i am not sure.

Reply:
Ok, then i will recreate the old one. 
Do you think there is any benefit of exposing the internal map?
I think It shouldn't be exposed, so if you agree i will just mark old getFiles() as deprecated and remove the one I added. In the test i will just use the getAllFiles method instead.

Reply:
Hmm. I don't think it needs to be public. It sure is nice but I wouldn't even recommend using it for anything but testing/debugging purposes

Reply:
Sorry for the late response,  please see the updated version